client/core/core.go

```diff
@@ -3084,6 +3082,13 @@ func (c *Core) checkUnpaidFees(dcrWallet *xcWallet) {
 // called if the client was shutdown after a fee was paid, but before it had the
```

Comment:
In `checkUnpaidFees` above, the `feeLock` and it's atomic accesses are now obsolete.  I believe it was not serving its intended purpose because `checkUnpaidFees` was not the only path to `verifyRegistrationFee`.

Reply:
Actually, maybe the `feeLock` is still required.  OK to leave it.